### DISCUSSION TITLE
Can someone explain why this app forwards traffic to some amazon machine?

### DISCUSSION THREAD

**Discussion by mihaicrisan** [Eugeny/tabby#9771] [General]
<img width="968" alt="Screenshot 2024-07-09 at 18 08 57" src="https://github.com/Eugeny/tabby/assets/12000295/23d35c01-482e-4341-a0f3-a2c93221d85c">

**Comment by specialk2hz** [comment]
Potentially the 'Enable Analytics' option, have you tried turning that off and checking again?

**Reply by mihaicrisan** [reply]
was the auto-update,  
but still, very curious why a port forward instead of a simple http at request start-up  to check for new versions?
